### PR TITLE
Color Palette For Drawing [ #24 ]( https://github.com/witseie-elen4010/2024-group-lab-003/issues/24 )

### LINKED ISSUE #24: As A Player I Can Draw In Many Different Colours

### PR DESCRIPTION
**Pull request by JosephKaplan613**
closes #24 
# Color Palette For Drawing

## Description
The players can now draw in different colors. To select a color the player toggles through a color palette. 

## Acceptance criteria
- The player can draw in different colors.
- When a player selects a new color to draw in, the new drawing stroke is in that color.

## Story Points

- 2  story points

### CODE REVIEW

File: src/views/drawing.html
Comment (akivalevitt) [inline src/views/drawing.html:1]: @JosephKaplan613 You have implemented the color picker well, however, for some reason you have two implementations, and one is commented out. Please remove this comment, it is unnecessary.

**Review by akivalevitt — CHANGES_REQUESTED** [review]
Functionality: Approved
The code introduces a color picker to change the pen color for a drawing application. The functionality is straightforward and effective, allowing users to select different colors via a simple GUI component.

Complexity: Approved
The complexity is minimal, which is appropriate given the functionality. It uses a basic event listener on a color picker to update a drawing tool's state. This simplicity is suitable for the context of a basic drawing application.

Tests: Denied
Tests have been provided however they seem to be failing, please fix this before I can merge to main branch

Naming: Approved
The function changeColor is aptly named, clearly indicating its purpose. Variable names like colorPicker are also descriptive. However, penColor should be declared with let or var if it's intended to be reassigned, to avoid scope issues.

Comments: Denied
You have left a large segment of commented out code, please remove this before we can merge this to the main branch.

Style: Approved
The JavaScript code follows a simple style, which is consistent and easy to follow. The HTML structure is also clear, though it has a minor typo with the closing </html> tag repeated twice, which should be corrected to ensure proper HTML document structure.

Documentation: Approved
N/A

**Comment by JosephKaplan613** [comment]
@akivalevitt  I have implemented the aforementioned changes. Please confirm if all is in order.

**Review by akivalevitt — APPROVED** [review]
Functionality: Approved
The code introduces a color picker to change the pen color for a drawing application. The functionality is straightforward and effective, allowing users to select different colors via a simple GUI component.

Complexity: Approved
The complexity is minimal, which is appropriate given the functionality. It uses a basic event listener on a color picker to update a drawing tool's state. This simplicity is suitable for the context of a basic drawing application.

Tests: Approved
Tests now pass, well done on fixing this

Naming: Approved
The function changeColor is aptly named, clearly indicating its purpose. Variable names like colorPicker are also descriptive. However, penColor should be declared with let or var if it's intended to be reassigned, to avoid scope issues.

Comments: Approved
Well done on removing the unnecessary commented out code.

Style: Approved
The JavaScript code follows a simple style, which is consistent and easy to follow. The HTML structure is also clear, though it has a minor typo with the closing tag repeated twice, which should be corrected to ensure proper HTML document structure.

Documentation: Approved
N/A

**Pull request merged**